storage/applicationinsights/src/main/java/zipkin/storage/applicationinsights/ApplicationInsightsSpanConsumer.java

```diff
@@ -68,6 +77,16 @@ public void accept(List<Span> spans) {
         }
         String res = gson.toJson(jsonElement);
         String msg = "{ \"Span\":" + res + "}";
+        //data model changes
+        telemetry.getContext().getOperation().setId(traceId);
+        telemetry.getContext().getOperation().setName(span.name);
+
+        for (Annotation annotation : span.annotations) {
+          if (annotation.value.equalsIgnoreCase(Constants.SERVER_RECV)) {
+            telemetry.trackDependency(Constants.SERVER_RECV, "request", new Duration(span.duration==null?0L:span.duration),
```

Comment:
`SERVER_RECV` means server received? When server receives - you need to track `request`. Dependency is when client send outgoing request.